app/models/usps_confirmation_code.rb

```diff
@@ -0,0 +1,16 @@
+class UspsConfirmationCode < ApplicationRecord
```

Comment:
I am definitely not in love with this name. It is a bit confusing since we have a `UspsConfirmation` model. Happy to change it if someone has a better idea.